gatsby-node.js

```diff
@@ -19,15 +18,25 @@ exports.onCreateWebpackConfig = ({ actions, getConfig }) => {
       ],
     },
     resolve: {
+      // Ensure node_modules are in the path for resolution
       modules: [path.resolve(__dirname, 'src'), 'node_modules'],
       alias: {
-        // light-weight fork of buble compiler from react-live team
         buble: path.resolve(__dirname, 'node_modules', '@philpl/buble'),
+        // Explicitly resolve @carbon packages to their node_modules path
+        '@carbon/react': path.resolve(__dirname, 'node_modules/@carbon/react'),
+        '@carbon/icons-react': path.resolve(__dirname, 'node_modules/@carbon/icons-react'),
+        '@carbon/styles': path.resolve(__dirname, 'node_modules/@carbon/styles'), // Keep for Sass if needed
       },
+      // Fallback for older module resolution patterns
+      fallback: {
+        'crypto': false, // Example of adding fallbacks if needed for older modules
+        'stream': false,
+        'path': require.resolve('path-browserify'), // Ensure path resolves correctly
+        // You might need more if other 'cannot resolve' errors appear for node built-ins
+      }
     },
   });
 
-  // Disable sourcemaps in production
```

Comment:
imo this is valuable as `devtool: false` doesn't really convey relation to sourcemaps